app/electron/main.ts

```diff
@@ -319,6 +344,30 @@ function getDefaultAppMenu(): AppMenu[] {
           role: 'forcereload',
           id: 'original-force-reload',
         },
+        {
+          label: i18n.t('Go To Home'),
+          role: 'homescreen',
+          id: 'original-home-screen',
+          click: () => {
+            mainWindow?.loadURL(startUrl);
+          },
+        },
+        {
+          label: i18n.t('Go Back'),
```

Comment:
We should add an accelerator: Alt+Left
(and whatever it is in Mac)

Reply:
done